pynamodb/attributes.py

```diff
@@ -406,6 +406,7 @@ def __init__(self, hash_key=False, range_key=False, null=None, default=None, att
                                            null=null,
                                            default=default,
                                            attr_name=attr_name)
+        self._is_raw_map = bool(type(self) == MapAttribute)
```

Comment:
I was under the impression that `==` is correct (and more pythonic?) for checking that the _value_ of two types is the same